docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -247,6 +247,7 @@ In addition to general platform and system errors, three errors are specific to
 |12004|The domain of the URL passed to `displayDialogAsync` is not trusted. The domain must be the same domain as the host page (including protocol and port number).|
 |12005|The URL passed to `displayDialogAsync` uses the HTTP protocol. HTTPS is required. (In some versions of Office, the error message returned with 12005 is the same one returned for 12004.)|
 |<span id="12007">12007</span>|A dialog box is already opened from this host window. A host window, such as a task pane, can only have one dialog box open at a time.|
+|12009|The user chose to ignore the dialog box. This can occur in the online versions of Office where users are given the option to not allow an Add-in to present a dialog.|
```

Comment:
Couple nits: "add-in" shouldn't be capitalized here. We could also condense that second sentence a bit. How does the following sound?
"This occurs in online versions of Office when users do not allow an add-in to present a dialog."

Reply:
I've updated the second sentence; resolving this conversation.